bigchaindb/core.py

```diff
@@ -43,19 +38,11 @@ def __init__(self, public_key=None, private_key=None, keyring=[], connection=Non
         its default value (defined in bigchaindb.__init__).
 
         Args:
-            public_key (str): the base58 encoded public key for the ED25519 curve.
-            private_key (str): the base58 encoded private key for the ED25519 curve.
-            keyring (list[str]): list of base58 encoded public keys of the federation nodes.
             connection (:class:`~bigchaindb.backend.connection.Connection`):
                 A connection to the database.
         """
-
         config_utils.autoconfigure()
 
-        self.me = public_key or bigchaindb.config['keypair']['public']
-        self.me_private = private_key or bigchaindb.config['keypair']['private']
-        self.nodes_except_me = keyring or bigchaindb.config['keyring']
```

Comment:
I did a grep search for any other mentions of `nodes_except_me` and found:
```text
tests/db/test_bigchain_api.py:559:            b.nodes_except_me.append(generate_key_pair()[1])
tests/db/test_bigchain_api.py:577:            assert tx['assignee'] in b.nodes_except_me
tests/db/test_bigchain_api.py:735:            'voters': b.nodes_except_me
```
I guess the associated tests aren't failing because they aren't being run.

Reply:
Yes.

Reply:
Removed the tests because they should be deprecated.